sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScanner.java

```diff
@@ -247,7 +250,10 @@ void addIncludedAndExcludedUrlsToContext() throws ClientApiException {
         String followRedirects = "false";
         for (String url : scanContext.getZapURLsIncludeSet()) {
             clientApiFacade.addIncludeUrlPatternToContext(scanContext.getContextName(), url);
-            clientApiFacade.accessUrlViaZap(url, followRedirects);
+            // cannot try to access wildcarded URLs
+            if (!url.contains(".*")) {
+                clientApiFacade.accessUrlViaZap(url, followRedirects);
+            }
```

Comment:
I think we should at least log the situation when the state "url.contains(".*") returns true